### ISSUE TITLE
macOS Package version numbering problem 

### ISSUE THREAD

**Issue by grahampugh** [s-andrews/SeqMonk#224]
Hi,

I've been tasked with packaging and deploying the macOS version of SeqMonk to managed computers in my organisation (ETH Zürich). A key facet of successful deployment and maintenance of managed software is version identification. 

I note that the latest version of SeqMonk.app is listed as 1.46.0 on the Babraham website, but the `CFBundleVersion` key in `SeqMonk.app/Contents/Info.plist` is still showing `0.25.0` as the version. Furthermore, the `CFBundleShortVersionString` key, which macOS Finder uses to determine the application version, is completely absent from the `Info.plist` file.

Please could you adapt your macOS application build process to ensure that the application version is properly identified in the `Info.plist` file?

Many thanks,
Graham Pugh

**Comment by s-andrews** [comment]
Hi Graham - thanks for reporting this.  It shouldn't be too hard to get that value updated - I don't think it's been touched since we started automating the builds - quite a long time ago.

Another issue you may hit if deploying to OSX is that there is currently an issue with the new security permissions introduced in Catalina which mean that default seqmonk installs can't access most of the filesystem (anything beyond the home directory).  Because our launch program is a script rather than a binary OSX fails to apply additional permissions correctly and the only current fix is a nasty kludge which isn't ideal.  You can see the details in #219 .  The eventual fix for this will be to write a binary launcher for OSX, but as I don't have direct access to Mac hardware during our enforced remote working I'm not sure when I'm going to be able to do this.  If you know of anyone familiar with C / C++ / Rust on OSX who would be able to help with what is probably a 10 line launch program we'd be very grateful for any assistance with this.

**Comment by s-andrews** [comment]
This is now fixed but the new versioning won't appear until the next release.

**Comment by grahampugh** [comment]
Thanks @s-andrews! Since the app is not signed, I was in the meantime able to "inject" the version into the app for distribution by our software deployment system. I did notice that there was another missing key which we needed to successfully package up the app; the `CFBundleIdentifier` key. Most apps have an identifier associated with them for various purposes, and it is required in order to sign it. Since it was absent, I made one up: `uk.ac.babraham.seqmonk`.
